orte/mca/grpcomm/direct/grpcomm_direct_component.c

```diff
@@ -55,7 +55,7 @@ static int direct_register(void)
     /* make the priority adjustable so users can select
      * direct for use by apps without affecting daemons
      */
-    my_priority = 1;
+    my_priority = 100;
```

Comment:
Do you want to leave some wiggle room here?  I.e., if you set this to 100, then it's not possible to override it -- even the user on the command line can't change priorities to something that would guarantee that direct wouldn't be chosen.